contributing/code/standards.rst

```diff
@@ -167,6 +167,24 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-conditions:
+
+Conditions
+----------
+
+Use `Yoda condition`_ when checking a variable against a value to avoid an
+accidental assignment inside the condition statements::
```

Comment:
@WouterJ, you mean in [structure](https://symfony.com/doc/current/contributing/code/standards.html#structure) section?

@Soullivaneuh, I think there is no possibility of accidental assignment in ```<``` and ```>```, so I agree with both of these.